### DISCUSSION TITLE
How to use multiple image/texture formats for spritesheet atlas?

### DISCUSSION THREAD

**Discussion by ddenisyuk** [pixijs/pixijs#10193] [Q&A]
Is it possible to define more than one spritesheet image format (png, webp, avif, dds, ktx, ...) for the same atlas in the manifest or spritesheet JSON?
As far as I know, only one image reference is allowed in the spritesheet JSON: https://github.com/pixijs/pixijs/blob/dev/packages/spritesheet/src/Spritesheet.ts#L50

**Comment by ddenisyuk** [comment]
@bigtimebuddy could you please assist?

**Reply by bigtimebuddy** [reply]
Please look at the documentation for [Assets](https://pixijs.download/dev/docs/PIXI.Assets.html) which has some information about loading multiple formats. the JSON format doesn't support this, so you should save multiple JSON files, one for each format variant.

**Reply by ddenisyuk** [reply]
Thank you for the answer. I was sure I had checked the documentation... 

But just to be sure, I got it right:

1. Can I use compressed textures (such as dds, ktx), like 'images/game-screen/game-screen@1x.dds.json'? I looked into the code, and the following line suggests not, but I might be looking in the wrong place -  https://github.com/pixijs/pixijs/blob/dev/packages/spritesheet/src/spritesheetAsset.ts#L19
2. Is the following statement applicable for spritesheet textures: "By default, we will prefer to load AVIF and WebP image files if you specify them. But if the browser doesn't support AVIF or WebP we will fall back to png and jpg."?
3. Also can you please confirm that this is the proper way to define it in the manifest.json:

```
{
      "name": "images/game-screen",
      "assets": [
        {
          "name": [
            "images/game-screen/game-screen.json"
          ],
          "srcs": [
            "images/game-screen/game-screen@1x.webp.json",
            "images/game-screen/game-screen@1x.avif.json",
            "images/game-screen/game-screen@1x.png.json",
            "images/game-screen/game-screen@0.5x.webp.json",
            "images/game-screen/game-screen@0.5x.png.json"
          ]
        }
      ]
    }
```

**Reply by ddenisyuk** [reply]
I conducted a few tests, and it seems that the texture resolution is not working as expected, nor is the spritesheet JSON when using imageFormat. It appears that these issues are related.
Please check https://github.com/pixijs/pixijs/issues/10215
